x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/index_parameter.tsx

```diff
@@ -30,7 +30,7 @@ export const IndexParameter = ({
       defaultMessage: 'Searchable',
     })}
     description={i18n.translate('xpack.idxMgmt.mappingsEditor.searchableFieldDescription', {
-      defaultMessage: 'Allow the field to be searchable.',
+      defaultMessage: 'Allow the field to be searched.',
```

Comment:
For the "Index options" help text, can we add a link to https://www.elastic.co/guide/en/elasticsearch/reference/master/index-options.html?

![image](https://user-images.githubusercontent.com/1238659/71038034-ca288f00-20d5-11ea-8389-e25744fdf5cd.png)

Reply:
Yep, I had planned to do that once https://github.com/elastic/kibana/pull/53060 is merged.

Reply:
Yep, I was planning on add it once https://github.com/elastic/kibana/pull/53060 is merged.